plugins/modules/redfish_config.py

```diff
@@ -171,6 +171,13 @@
     type: dict
     default: {}
     version_added: '7.5.0'
+  power_restore_policy:
+    required: false
+    description:
+      - The desired power state of the system when power is restored after a power loss.
+    type: str
+    default: ''
```

Comment:
Out of curiosity, why is the default an empty string? Is this parameter not required when `SetPowerRestorePolicy` is used?

Reply:
No, empty string there makes no sense, I just thought that if `[]` is appropriate for `boot_order` then an empty string might be a good default too. Now I see how much better it is with the `choices` and no default set.